install_config/install/prerequisites.adoc

```diff
@@ -805,3 +805,61 @@ Following the installation process, you can configure {product-title} for
 xref:../../install_config/configuring_aws.adoc#install-config-configuring-aws[AWS],
 xref:../../install_config/configuring_openstack.adoc#install-config-configuring-openstack[OpenStack], or
 xref:../../install_config/configuring_gce.adoc#install-config-configuring-gce[GCE].
+
+[[prereq-containerized-glusterfs-considerations]]
+=== Containerized GlusterFS Considerations
+
+If you choose to configure
+xref:../../install_config/install/advanced_install.adoc#advanced-install-containerized-glusterfs-persistent-storage[containerized GlusterFS persistent storage] for your cluster, or if you choose to
+configure a
+xref:../../install_config/install/advanced_install.adoc#advanced-install-glusterfs-backed-registry[containerized GlusterFS-backed OpenShift Container Registry], you must consider the following prerequisites.
+
+[[prereq-glusterfs-storage-nodes]]
+==== Storage Nodes
+
+To use containerized GlusterFS persistent storage:
+
+- Each storage node is recommended to have at least 1 raw block device with least
+100 GB available.
+
+To run a containerized GlusterFS-backed OpenShift Container Registry:
+
+- A minimum of 3 storage nodes must be available for 3-way replication.
+- Each storage node must have at least 1 raw block device with at least 10 GB of
+free storage. 
+
+[IMPORTANT]
+====
+While containerized GlusterFS persistent storage can be configured and deployed
+on the same {product-title} cluster as a containerized GlusterFS-backed
+registry, their storage should be kept separate from each other and also
+requires additional storage nodes. For example, if both are configured, a total
+of 6 storage nodes would be needed: 3 for the registry and 3 for persistent
+storage. This limitation is based on Heketi's ability to only manage a single
+storage pool at this time.
+====
+
+[[prereq-glusterfs-software-components]]
+==== Required Software Components
+
+ifdef::openshift-enterprise[]
+For any RHEL (non-Atomic) storage nodes, the following RPM respository must be
+enabled:
+
+----
+# subscription-manager repos --enable=rh-gluster-3-for-rhel-7-server-rpms
+----
+endif::[]
+
+The *glusterfs-fuse* RPM package, which provides `mount.glusterfs`, must be
```

Comment:
glusterfs-fuse or glusterfs-client package? - Not sure which is more official/appropriate, I believe in the past (for RPM based installs) we always installed the glusterfs-client (i.e.   yum install glusterfs-client -y), that automatically gives you fuse and some other stuff.